### PR TITLE
add disable prop to contact-item builder

### PR DESCRIPTION
**Pull request by malachico**
### 🔦 Summary
...

### ✅ Checklist
- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by shlomitc** [comment]
please add also an example in the docs so it can be reviewed visually

File: src/ContactItemBuilder/ContactItemBuilder.spec.js [hunk 1]
```diff
@@ -26,5 +26,17 @@ describe('item picker option builder', () => {
     expect(driver.getSubtitle()).toEqual(subtitle);
   });
 
+  it('should return item with disabled prop', () => {
+    const component = (
+      <ContactItem title={title} subtitle={subtitle} disabled />
+    );
+    expect(component.props.disabled).toBeTruthy();
```
Comment (hepiyellow) [inline src/ContactItemBuilder/ContactItemBuilder.spec.js]: This test is only checking that ContactItem is a JSX item. And prop you pass that way will be available on `component.prop`.

You can calling the method `contactItemBuilder()`?
and expect the result to be an object with the `disabled` property.

File: src/ContactItemBuilder/ContactItemBuilder.spec.js [hunk 2]
```diff
@@ -26,5 +26,17 @@ describe('item picker option builder', () => {
...
+  });
+
+  it('should return item without disabled prop when prop is not passed', () => {
+    const contactItem = <ContactItem title={title} subtitle={subtitle} />;
```
Comment (hepiyellow) [inline src/ContactItemBuilder/ContactItemBuilder.spec.js]: Same here. You should call `contactItemBuilder()`

File: src/ContactItemBuilder/ContactItemBuilder.js [hunk 3]
```diff
@@ -46,10 +46,18 @@ ContactItem.propTypes = {
   title: PropTypes.string.isRequired,
   subtitle: PropTypes.string,
   imageUrl: PropTypes.string,
+  disabled: PropTypes.bool,
```
Comment (hepiyellow) [inline src/ContactItemBuilder/ContactItemBuilder.js]: You added `disabled` prop to `<ContactItem/>` but you didn't do anything with it.
Please remove it.

**Pull request merged**